editor/src/core/shared/github.ts

```diff
@@ -355,66 +367,61 @@ export async function updatePullRequestsForBranch(
 
   dispatch([updateGithubOperations(operation, 'add')], 'everyone')
 
-  const url = urljoin(
-    UTOPIA_BACKEND,
-    'github',
-    'branches',
-    githubRepo.owner,
-    githubRepo.repository,
-    'branch',
-    branchName,
-    'pullrequest',
-  )
+  try {
+    const url = urljoin(
+      UTOPIA_BACKEND,
+      'github',
+      'branches',
+      githubRepo.owner,
+      githubRepo.repository,
+      'branch',
+      branchName,
+      'pullrequest',
+    )
 
-  const response = await fetch(url, {
-    method: 'GET',
-    credentials: 'include',
-    headers: HEADERS,
-    mode: MODE,
-  })
+    const response = await fetch(url, {
+      method: 'GET',
+      credentials: 'include',
+      headers: HEADERS,
+      mode: MODE,
+    })
 
-  if (response.ok) {
-    const responseBody: GetBranchPullRequestResponse = await response.json()
+    if (response.ok) {
+      const responseBody: GetBranchPullRequestResponse = await response.json()
 
-    switch (responseBody.type) {
-      case 'FAILURE':
-        dispatch(
-          [
+      switch (responseBody.type) {
+        case 'FAILURE':
+          return [
             showToast(
               notice(
                 `Error when listing pull requests for branch: ${responseBody.failureReason}`,
                 'ERROR',
               ),
             ),
-          ],
-          'everyone',
-        )
-        break
-      case 'SUCCESS':
-        dispatch(
-          [
+          ]
+          break
```

Comment:
not needed

Reply:
Fixed in cc6fda6c71313001c3045ec5dd87329b0039f5b4.